README.md

```diff
@@ -154,15 +190,15 @@ Write Ruby code that will create an instance of a person.
 Your answer...
 
 ```ruby
-Replace this with your answer
+Person.new()
```

Comment:
👍 no need to add parens if no arguments. Also, want to use `create` and actually provide name and age values in order to persist